### PR TITLE
Updated maintainer list and glossary (#355)

### LINKED ISSUE #355: Add glossary terms
The [glossary](https://datacarpentry.org/sql-ecology-lesson/reference.html#glossary) for this lesson is empty, and that page currently has a big FIXME header on it. Need to settle on some terms and fill them in, probably from the [Carpentries Glossary](https://glosario.carpentries.org/).

### PR DESCRIPTION
**Pull request by adamancer**
Resolves #355. @jd-foster, I'd be curious to hear your thoughts on this. Some of the definitions feel a little technical. I think that's probably okay because they're not part of the main lesson flow but I could be convinced otherwise.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,16 +11,15 @@ on how to contribute updates, bug fixes, or other corrections.
...
 - James Foster ([@jd-foster](https://github.com/jd-foster))
-- Sathya Pandalai ([@spandalai](https://github.com/spandalai))
+- Adam Mansur ([@adamancer](https://github.com/adamancer))
 
 ### Past maintainers
 
+- Katy Felkner ([@katyfelkner](https://github.com/katyfelkner))
 - Christina Koch ([@ChristinaLK](https://github.com/ChristinaLK))
+- Paula Andrea Martinez ([@orchid00](https://github.com/orchid00))
+- Sathya Pandalai ([@spandalai](https://github.com/spandalai))
```
Comment (jd-foster) [inline README.md:22]: This line can be removed (I never heard from them.)

File: learners/reference.md [hunk 2]
```diff
@@ -1,15 +1,105 @@
...
+[missing value](https://glosario.carpentries.org/en/#missing_value)
+:   A special value such as [null](#null) or NA used to indicate the
+    absence of data. Missing values can signal that data was not
+    collected or that the data did not exist in the first place (e.g.,
+    the middle name of someone who does not have one).
+
+[null](https://glosario.carpentries.org/en/#null)
+:   A special value used to represent a missing object.
+
+[record](https://glosario.carpentries.org/en/#record)
```
Comment (jd-foster) [inline learners/reference.md:63]: Seems this needs a special notation

Comment (jd-foster) [inline learners/reference.md:63]:
```suggestion
[[__record__]{#record}](https://glosario.carpentries.org/en/#record)
```

**Comment by jd-foster** [comment]
This is great, BTW.

File: learners/reference.md (hunk 2 above)
Comment (adamancer) [reply learners/reference.md]: Fixed the links, thanks! This is what I get for not actually building the lesson.

File: learners/reference.md [hunk 3]
```diff
@@ -7,87 +7,87 @@ title: Reference
 *The definitions below are modified from the [Carpentries
 Glosario](https://glosario.carpentries.org/) (CC-BY-4.0)*
 
-[aggregation](https://glosario.carpentries.org/en/#aggregation)
+[[**aggreation**]{#aggreation}](https://glosario.carpentries.org/en/#aggreation)
```
Comment (jd-foster) [inline learners/reference.md:10]:
```suggestion
[[**aggregation**]{#aggregation}](https://glosario.carpentries.org/en/#aggregation)
```

File: learners/reference.md [hunk 4]
```diff
@@ -1,15 +1,105 @@
...
+[[**null**]{#null}](https://glosario.carpentries.org/en/#null)
+:   A special value used to represent a missing object.
+
+[[**record**]{#record}](https://glosario.carpentries.org/en/#record)
+:   A group of related values that are stored together. A record may be
+    represented as a tuple or as a row in a [table](#table); in the
+    latter case, every record in the table has the same
+    [fields](#field).
+
+[[**relational_database**]{#relational_database}](https://glosario.carpentries.org/en/#relational_database)
```
Comment (jd-foster) [inline learners/reference.md:69]:
```suggestion
[[**relational database**]{#relational_database}](https://glosario.carpentries.org/en/#relational_database)
```

**Comment by jd-foster** [comment]
The markdown parser seems to not like tags with underscores e.g. `#relational_database`
Changing to hyphen e.g `#relational-database` seems to be fine.
It renders either way. Otherwise looks good.

File: learners/reference.md (hunk 4 above)
Comment (adamancer) [reply learners/reference.md]: Fixed these, thanks!

**Pull request merged**